inc/saiswitch.h

```diff
@@ -490,6 +495,32 @@ typedef enum _sai_switch_tunnel_attr_t
      */
     SAI_SWITCH_TUNNEL_ATTR_VXLAN_UDP_SPORT,
 
+    /**
+     * @brief Tunnel UDP source port base for a calculated hash range
+     *
+     * Value of the fixed portion of the UDP source port's MSB
+     *
+     * @type sai_uint16_t
+     * @flags CREATE_AND_SET
+     * @isvlan false
+     * @default 0
+     * @validonly SAI_SWITCH_TUNNEL_ATTR_TUNNEL_TYPE == SAI_TUNNEL_TYPE_VXLAN and SAI_SWITCH_TUNNEL_ATTR_TUNNEL_VXLAN_UDP_SPORT_MODE == SAI_TUNNEL_VXLAN_UDP_SPORT_MODE_USER_DEFINED_HASH
+     */
+    SAI_SWITCH_TUNNEL_ATTR_VXLAN_UDP_SPORT_BASE,
+
+    /**
+     * @brief Tunnel UDP source port hash bit length
+     *
+     * Number of LSB reserved for the calculated hash value
+     *
+     * @type sai_uint16_t
```

Comment:
probably can be uint8_t instead of uint16_t, as we need to describe a value between 0 to 16